core/vcsl/vcsl_coordinate_system.cxx

```diff
@@ -24,14 +25,14 @@ vcsl_axis_sptr vcsl_coordinate_system::axis(int i) const
 // REQUIRE: v.size()==dimensionality()
 //---------------------------------------------------------------------------
 vnl_vector<double>
-vcsl_coordinate_system::from_cs_to_standard_units(const vnl_vector<double> &v) const
+vcsl_coordinate_system::from_cs_to_standard_units(const vnl_vector<double> & v) const
 {
   vnl_vector<double> result(v.size());
 
-  int j=0;
+  int                                         j = 0;
```

Comment:
@hjmjohnson I appreciate your aim for consistency, but you know I really don't like this excessive alignment 😢

Reply:
@N-Dekker Your objection is noted and understood.  As discussed previously, there is a slight preference from the active development community to enforce this subjective opinion that this consistency improves "readability".

In this somewhat outlier case, I agree that the result is not optimal.


Thank you for your careful consideration.

Reply:
For those who missed it, here is again the link to the talk by Kevlin Henney, hoping to clarify my point of view on this issue: [Seven Ineffective Coding Habits of Many Programmers ](https://www.youtube.com/watch?v=ZsHMHukIlJY)

Reply:
Thanks @N-Dekker.  That is an interesting video with many great aspirational ideas/themes.  I strongly believe that the compromise solution converged on with the automated formatting tool applied here is a meaningful and helpful improvement to the codebase.

Reply:
I'm in agreement that consistency is in general a good thing, but I really don't like this alignment.  Mostly because it looks very ugly to me, but, more objectively, doesn't it also introduce unnecessary dependencies between adjacent lines of code, potentially leading to commit noise?

Reply:
@N-Dekker @decrispell   I agree with you on this one case, but strongly believe that the overall code improvements represent a net benefit to the readability and maintenance burden. 

I have an update that fixes this case, and could easily be applied to other cases as well.  Improving the consistency of variables that are related to each other by having a block alignment of those variables is often attributed to improving understanding of code (clearly not everyone agrees).

Some benefits of the automatic formatting in this PR are:  
   - [X] It is the same as the ITK toolkit where many people carefully critiqued and discussed the pros & cons of each formatting decision.  Of course it is not universally liked, but it has been rigorously vettted.
  - [X] We don't need to re-invent mechansims for application
 - [X] IT IS AUTOMATED!  And many IDE's development environments respect this formatting enforcement.  That can not be understated how much burden can be removed from developer efforts when a fully automated tool can be used.
 - [X] IT IS AUTOMATED! Just stating that important fact again :).   This is also a con, because in a few cases the rules enforcement is not easy to disable.  They can be disabled with formatting 
The code between a comment // clang-format off or /* clang-format off */ up to a comment // clang-format on or /* clang-format on */

Reply:
I don't have any experience with automated code formatting, but I'm certainly open to it if the tools are available and reliable.

Is doing alignment like this somehow needed to achieve automated formatting though?   I'd like to see the discussions that led to other projects adopting it.  The only one I see (ITK, linked in the original PR comment) included a poll where the majority of developers were "strongly opposed" to it.

Reply:
If the votes are weighted by contribution level, the poll was not so clear cut. I still think the majority was in favor of no alignment. But as Hans was the one doing the refactoring, he chose to keep the alignment style. That is the benefit of conducting the effort, over just rooting for some choice or the other 😄